.storybook/recipes/GlobalHeader/GlobalHeader.tsx

```diff
@@ -26,42 +26,56 @@ export const GlobalHeader: React.FC<Props> = ({ className, ...other }) => {
 
   const toggleMenu = () => {
     setisActive(!isActive);
+    if (isActive) {
+      document.body.classList.remove('eds-is-disabled');
+    } else {
+      document.body.classList.add('eds-is-disabled');
+    }
   };
 
   const componentClassName = clsx(styles['global-header'], className, {
     [styles['is-active']]: isActive,
   });
   return (
     <Header behavior="sticky" className={componentClassName} {...other}>
-      <div className={styles['global-header__header']}>
-        <Logo href="#" />
+      <Logo className={styles['global-header__logo']} href="#" />
 
-        <Button
-          className={styles['global-header__menu-button']}
-          variant="bare"
-          text={isActive ? 'Close' : 'Menu'}
-          iconPosition="before"
-          iconName={isActive ? 'x' : 'menu'}
-          inverted={true}
-          onClick={toggleMenu}
-        />
-      </div>
+      <Button
+        className={styles['global-header__menu-button']}
+        variant="bare"
+        text={isActive ? 'Close' : 'Menu'}
+        iconPosition="before"
+        hideText={true}
```

Comment:
@ifrost1  instead of adding the `hideText` prop (which we've already decided to remove in https://github.com/chanzuckerberg/edu-design-system/pull/889/files#r822192445) -- could this be
```suggestion
        aria-label={isActive ? 'Close' : 'Menu'}
        iconPosition="before"
```
? I don't think this is dependent on Diedra's upcoming button work